### DISCUSSION TITLE
disable magnetism

### DISCUSSION THREAD

**Discussion by DevIA05** [pubpub-zz/ppInk#316] [Q&A]
Hello,
How to disable magnetism when you want to make two figures (rectangle for example) next to each other ? when I draw a second rectangle below a first rectangle, the second rectangle is drawn from the bottom of the first rectangle and not where the cursor is.

**Comment by pubpub-zz** [comment]
You have 2 options : magnetic effect can be completly turned off with the magnet icon, on you can depress ctrl+shift. ctrl or shift only will disable it partially

**Comment by DevIA05** [comment]
thank for your help.
